packages/api/webpack.config.js

```diff
@@ -80,7 +84,8 @@ module.exports = {
   externals: [
     'aws-sdk',
     'electron',
-    { formidable: 'url' }
+    { formidable: 'url' },
+    { fsevents: "require('fsevents')" }
```

Comment:
For this following error, solution is [here](https://github.com/yan-foto/electron-reload/issues/71):
ERROR in /Users/jhliu/cumulus/node_modules/fsevents/fsevents.node 1:0
Module parse failed: Unexpected character '�' (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders